src/aind_metadata_service/sharepoint/client.py

```diff
@@ -44,6 +47,11 @@ class ProcedureType(Enum):
         INJECTION = "Injection"
         INJ = "INJ"
         OPTIC_FIBER_IMPLANT = "Optic Fiber Implant"
+        HP_TRANSCRANIAL = "HP Transcranial (for ISI)"
```

Comment:
@saskiad Is there anything valuable to keep track of out of these values? Looks mostly like project codes so we can probably ignore. The fact that it's a transcranial craniotomy is maybe already covered by craniotomy size?

Reply:
I'll actually be mapping HP Transcranial to headplant in another PR

Reply:
yes, I think these values will help us to map to protocols. There are also hidden parameters associated with these procedures that don't show up in the request form (eg. craniotomy coordinates) that are determined by the type of procedure. 
Some of the items on this list might be obsolete for our purposes, but we can prune them as we need